index.html

```diff
@@ -6,11 +6,17 @@
     <title>Student Grade Table</title>
 
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" />
+
     
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
 
     <script type="text/javascript" src="component/app.js"></script>
     <script type="text/javascript" src="component/page-header.js"></script>
+    <script type="text/javascript" src="component/grade-form.js"></script>
     <script type="text/javascript" src="main.js"></script>
     <script type="text/javascript" src="component/grade-table.js"></script>

```

Comment:
In general, if a `<script>` depends on another `<script>`, you want to include the one it depends on first. Since `main.js` uses the classes defined in every other `<script>` you want to include `main.js` last. Also, since `app.js` uses the other components indirectly (they're passed into its `constructor`) it should be after them (but still before `main.js`).